client/connect-account-page/index.tsx

```diff
@@ -22,14 +22,14 @@ import { addQueryArgs } from '@wordpress/url';
 import { recordEvent } from 'tracks';
 import Page from 'components/page';
 import BannerNotice from 'components/banner-notice';
-import PaymentMethods from './payment-methods';
```

Comment:
@dpaun1985 Do you think we’re using the old component elsewhere? I couldn’t find any references to it, so maybe it’s a good idea to delete the obsolete code. What do you think?

Reply:
removed in [11a0695](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/11a06957c1895d9c07806393f86fad26f6324ce3) . thank you!